boards/common/weact-f4x1cx/include/periph_conf.h

```diff
@@ -36,7 +36,17 @@
 #define CLOCK_HSE               MHZ(25)
 
 #include "periph_cpu.h"
+
+#if defined(CPU_LINE_STM32F401xC) || defined(CPU_LINE_STM32F401xE)
+#define CONFIG_CLOCK_PLL_M      (25)
+#define CONFIG_CLOCK_PLL_N      (336)
```

Comment:
You could move these particular defines directly in `f2f4f7/cfg_clock_default_84.h`, like it's done in `f2f4f7/cfg_clock_default_96.h`.
There's also no reason to have these defines non overridable: we should allow other possible configurable combinations for the PLL clock parameters.

Reply:
Thank you for the hint, that makes it much cleaner.